### ISSUE TITLE
Files section and field ignore image preview settings

### ISSUE THREAD

**Issue by moritzebeling** [getkirby/kirby#3226]
**Describe the bug**  
The `files` section and field, both ignore the `image.cover: false` option. 

**To Reproduce**  
1. Create a page with a files section and a files field, e.g.
```yaml
sections:

  uploads:
    type: files
    image:
      cover: false

  fields:
    type: fields
    fields:

      gallery:
        type: files
        image:
          cover: false
```
2. Upload and select some images
3. You will see, that both the section and the field have cropped images

**Expected behavior**  
The cropping or fitting of the image should follow the `image.cover` option.

This topic is important for image editors to instantly see the image orientation and aspect ratio.

**Screenshots**  
<img width="384" alt="Bildschirmfoto 2021-03-30 um 09 16 28" src="https://user-images.githubusercontent.com/39742729/112948675-a22d3580-9138-11eb-89bb-e96943bde01b.png">

**Kirby Version**  
3.5.3.1

**Device**  
macOS, safari

**Comment by afbora** [comment]
I guess `cover` and `ratio` feature for `cards` layout 🤔

**Comment by moritzebeling** [comment]
You mean that this is ignored for the list view?
But I somehow remember, that it once worked, maybe 1 year ago? I am pretty sure, that I once saw it...

However, I think it would be helpful to (have the option to) immediately see if an image is landscape or portrait.

**Comment by afbora** [comment]
> You mean that this is ignored for the list view?

Yes it looks like that.

> But I somehow remember, that it once worked, maybe 1 year ago? I am pretty sure, that I once saw it...

Hmm, I'll test on old versions.

**Comment by moritzebeling** [comment]
Or did I maybe saw it in the `pages` field/section?

If I am wrong and this was never a feature, excuse moi – I hereby would like to suggest it :)
In that case, feel free to move this issue to the `ideas` repo. Thank you!

**Comment by afbora** [comment]
@moritzebeling I've tested on Kirby 3.3.3 and `cover: false` works on `list` layout as your said:

![screen-capture-980-Ocean - Mægazine-localhost](https://user-images.githubusercontent.com/3393422/114002722-cb02a880-9865-11eb-979f-f50149c470dc.png)


I don't know if this is a feature that was added and intended in later versions. Pinging @getkirby/kirby-staff to get details.

**Comment by lukasbestle** [comment]
@afbora Could you please check in which exact version this feature broke? Then we can track down which PR and exact code change caused it. Looks like a regression to me.

**Comment by afbora** [comment]
@lukasbestle I guess i found the source of issue.

I can swear it worked in 3.3.3, but it didn't work today. I've tried the older versions too.
In 3.1.4 `covers: false` works, but not in 3.2.0.

In 3.1.4, the images in the list layout in the panel are larger and the ratio is the same as the original image. But in 3.2.0, the size of the images in the list layout is always 38x38px. Then I followed and reached the following commit:

https://github.com/getkirby/kirby/commit/0efd1252e863e8d893803af67d3b3772d0262f15#diff-54c2ceeba646ff0a67d792ee0def36ea9e0e3e9b20693f28d49e98033eff503f

I think this issue about images settings overwritten in list layout in new `ModelWithContent::panelImage()` method:

https://github.com/getkirby/kirby/blob/3.2.0/src/Cms/ModelWithContent.php#L328-L343

**Possible solution**

The solution I can think of is to add a control if not `cover: false`: `$settings['cover'] !== false` 🤔
